VRCFaceTracking/Logger.cs

```diff
@@ -11,31 +11,60 @@ public static class Logger
 
         public static void Msg(string msgStr)
         {
-            var formattedStr = "[" + DateTime.Now.ToString("HH:mm:ss") + "] " + msgStr;
-            
-            ConsoleOutput.Add(new Tuple<string, string>(formattedStr, "White"));
-        
-            Console.WriteLine(formattedStr);
+            var timeStamp = "[" + DateTime.Now.ToString("HH:mm:ss") + "] ";
+            var formattedStr = timeStamp + msgStr;
+
+            try
+            {
+                ConsoleOutput.Add(new Tuple<string, string>(formattedStr, "White"));
+                Console.WriteLine(formattedStr);
+            }
+            catch (Exception e)
+            {
+                ConsoleOutput.Add(new Tuple<string, string>(formattedStr + "Message intercepted while writing. " + e, "White"));
+                Console.WriteLine(formattedStr);
+            }
```

Comment:
Just out of curiosity, how often does this happen and what are the conditions for the catch to be triggered?

Reply:
I've seen it happen when switching out external modules and reinitializing while the logger is handling a message from an external module. I believe the changes you've made in a different commit to the logger are a more thorough fix to that issue.

Reply:
Gonna mention https://github.com/benaclejames/VRCFaceTracking/issues/121 for future ref.